src/gen_meta.ml

```diff
@@ -32,7 +33,10 @@ module Pub_name = struct
   let to_string t = String.concat ~sep:"." (to_list t)
 end
 
-let string_of_deps deps = String.Set.to_list deps |> String.concat ~sep:" "
+let string_of_deps deps =
+  Lib_name.Set.to_list deps
+  |> List.map ~f:Lib_name.to_string
+  |> String.concat ~sep:" "
```

Comment:
We could also add `Lib_name.Set.to_string_set` that would be the identity. That would avoid the extra `List.map`

Reply:
I think we'd need `to_string_list` here.